src/components/TextInput/TextInput.js

```diff
@@ -206,6 +208,20 @@ class TextInput extends React.Component<Props, State> {
     });
   };
 
+  handleMultilineFocus = () => {
+    if (!this.state.isFocused) {
+      this.rnInput.current.focus();
+      this.setState({
+        isFocused: false,
```

Comment:
Just checking, isn't `state.isFocused` already `false`? Maybe this should set it to `true`?